mingw-w64-git/PKGBUILD

```diff
@@ -144,9 +144,20 @@ build() {
   CPPFLAGS="$(echo "$CPPFLAGS" |
 	sed 's/-D__USE_MINGW_ANSI_STDIO\(=[0-9]*\)\?//')"
 
+  if [ "${IS_ARM64}" = "true" ]; then
+    echo "Found IS_ARM64 variable. We'll do some special handling in git-wrapper.c for this architecture."
+    echo "All packages are mingw32 until native arm64 support is available, but the Git wrapper should point to the native Git ARM64 binary."
+    rm -f *.res &&
+    IS_ARM64=true make -n -f ../mingw-w64-git.mak git-wrapper.exe
```

Comment:
I added -n to get some verbose output, so that I could verify if IS_ARM64 is working correctly. Happy to remove that prior to merging this PR